osdep/mac/meson.build

```diff
@@ -1,8 +1,6 @@
 # custom swift targets
 bridge = join_paths(source_root, 'osdep/mac/app_bridge_objc.h')
-header = join_paths(build_root, 'osdep/mac/swift.h')
 module = join_paths(build_root, 'osdep/mac/swift.swiftmodule')
```

Comment:
seems like module isn't used either anymore.

Reply:
seems to be used here https://github.com/mpv-player/mpv/blob/8eea5902baeaadf40cc3956edba9e3e17f9d9a3f/osdep/mac/meson.build#L71

Reply:
i am just being blind.